{"time":"2024-11-14T02:10:31.372978+01:00","level":"INFO","msg":"using version","core version":"0.18.5"}
{"time":"2024-11-14T02:10:31.3735078+01:00","level":"INFO","msg":"created symlink","path":"C:\\Users\\barbo\\brain classifier repo\\brain_classifier\\wandb\\run-20241114_021031-othh6wsy\\logs\\debug-core.log"}
{"time":"2024-11-14T02:10:31.8917337+01:00","level":"INFO","msg":"created new stream","id":"othh6wsy"}
{"time":"2024-11-14T02:10:31.8922386+01:00","level":"INFO","msg":"stream: started","id":"othh6wsy"}
{"time":"2024-11-14T02:10:31.8922386+01:00","level":"INFO","msg":"sender: started","stream_id":"othh6wsy"}
{"time":"2024-11-14T02:10:31.8922386+01:00","level":"INFO","msg":"writer: Do: started","stream_id":{"value":"othh6wsy"}}
{"time":"2024-11-14T02:10:31.8922386+01:00","level":"INFO","msg":"handler: started","stream_id":{"value":"othh6wsy"}}
{"time":"2024-11-14T02:10:32.1918178+01:00","level":"INFO","msg":"Starting system monitor"}
{"time":"2024-11-14T02:10:37.3330288+01:00","level":"INFO","msg":"Stopping system monitor"}
{"time":"2024-11-14T02:10:37.3341334+01:00","level":"INFO","msg":"Stopped system monitor"}
{"time":"2024-11-14T02:10:38.3346711+01:00","level":"INFO","msg":"handler: operation stats","stats":{"operations":[{"desc":"updating run config","runtime_seconds":0.9984072}],"total_operations":1}}
{"time":"2024-11-14T02:10:47.8156715+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/files/barbonigiorgia-sapienza-universit-di-roma/graph-classification-graphs_sc_combined/othh6wsy/file_stream\": unexpected EOF"}
{"time":"2024-11-14T02:10:52.048131+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": read tcp 192.168.1.10:58248->35.186.228.49:443: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2024-11-14T02:10:52.048131+01:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": read tcp 192.168.1.10:58248->35.186.228.49:443: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."}
{"time":"2024-11-14T02:10:55.3608935+01:00","level":"INFO","msg":"fileTransfer: Close: file transfer manager closed"}
{"time":"2024-11-14T02:10:56.6123483+01:00","level":"INFO","msg":"stream: closing","id":"othh6wsy"}
{"time":"2024-11-14T02:10:56.6123483+01:00","level":"INFO","msg":"handler: closed","stream_id":{"value":"othh6wsy"}}
{"time":"2024-11-14T02:10:56.6123483+01:00","level":"INFO","msg":"sender: closed","stream_id":"othh6wsy"}
{"time":"2024-11-14T02:10:56.6123483+01:00","level":"INFO","msg":"writer: Close: closed","stream_id":{"value":"othh6wsy"}}
{"time":"2024-11-14T02:10:56.6129767+01:00","level":"INFO","msg":"stream: closed","id":"othh6wsy"}
